astropy/io/votable/tests/test_table.py

```diff
@@ -304,13 +305,8 @@ def test_read_write_votable_parquet(tmp_path):
         format="votable.parquet",
     )
 
-    print(os.listdir(tmp_path))
-    print(filename)
-    print("============")
-    with open(filename, 'r') as f:
-        print(f.read())
-    print("============")
     # Open created VOTable with Parquet serialization
+    warnings.simplefilter("always", ResourceWarning)
```

Comment:
Can this at least be in a context manager? I am afraid this would change the global warning state of the entire test suite without. Thanks!

Reply:
oh, sure, sorry for the oversight.